src/test/java/org/prebid/server/bidder/colossus/ColossusBidderTest.java

```diff
@@ -212,10 +235,11 @@ private static BidRequest givenBidRequest(
 
     private static Imp givenImp(Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer) {
         return impCustomizer.apply(Imp.builder()
-                .id("123"))
+                        .id("123"))
```

Comment:
Why does this change exist?

Reply:
When do format `CTRL + ALT  + L` and it's occur